### ISSUE TITLE
Generate doesn't work using the example

### ISSUE THREAD

**Issue by Blatty** [paljs/prisma-tools#164]
Hello,

First, thanks a lot for your hard work on these tools, they are very helpful!

I just tried to generate a CRUD system using `pal g` for the `nexus-plugin-prisma` but it looks like it's not working (I've been using the example):

```prisma
// schema.prisma

datasource postgresql {
  url      = env("DATABASE_URL")
  provider = "postgresql"
}

generator client {
  provider = "prisma-client-js"
}

model User {
  id        String   @id @default(cuid())
  email     String   @unique
  birthDate DateTime
  posts     Post[]
}

model Post {
  id     String @id @default(cuid())
  author User[]
}
```

```ts
// pal.js
module.exports = {
  backend: {
    generator: 'nexus-plugin-prisma',
  },
};
```

```bash
$ Prisma Schema loaded from prisma/schema.prisma
◢ Generating your files    SyntaxError: Unexpected token, expected "," (30:63)
      28 |                     },
      29 |                     async resolve(_root, args, ctx) {
    > 30 |                       return ctx.prisma.user.findFirst(argsas any)
         |                                                               ^
      31 |                     },
      32 |                   })
      33 | t.crud.users({ filtering: true, ordering: true })
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

```bash
$ pal --version
@paljs/cli/2.9.3 darwin-x64 node-v12.20.0
```

Am I missing anything?

Cheers

**Comment by AhmedElywa** [comment]
No you are right it's issue will fix soon

**Comment by mohsin51** [comment]
+1

**Comment by AhmedElywa** [comment]
This issue fixed in the last release `v2.10`, please upgrade to this version.
